composer.json

```diff
@@ -93,6 +91,7 @@
         "symfony/debug-bundle": "^3.0",
         "symfony/dom-crawler": "^3.0",
         "symfony/monolog-bundle": "^3.0",
+        "symfony/monolog-bridge": "^3.0",
```

Comment:
else it install monolog-bridge 2.7 which is not really compatible in the tests with http-kernel 3